.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -133,7 +131,8 @@ def get_input(question, default=""):
             'authentication_method': 1,
             'institution_name' : '',
             'username_change_text' : 'Submitty welcomes individuals of all ages, backgrounds, citizenships, disabilities, sex, education, ethnicities, family statuses, genders, gender identities, geographical locations, languages, military experience, political views, races, religions, sexual orientations, socioeconomic statuses, and work experiences. In an effort to create an inclusive environment, you may specify a preferred name to be used instead of what was provided on the registration roster.',
-            'institution_homepage' : ''}
+            'institution_homepage' : '',
+            'timezone' : tzlocal.get_localzone()}
```

Comment:
you probably want to do `tzlocal.get_localzone().zone()` here as `tzlocal.get_localzone()` returns a TzInfo object. Example:
```
>>> import tzlocal
>>> tzlocal.get_localzone()
<DstTzInfo 'America/New_York' LMT-1 day, 19:04:00 STD>
>>> tzlocal.get_localzone().zone
'America/New_York'
```